webserver/views/datasets.py

```diff
@@ -230,7 +231,7 @@ def create():
 
 
 @datasets_bp.route("/import", methods=("GET", "POST"))
-@login_required
+@api_login_required
```

Comment:
You need to check the functionality of these methods before making changes like this - this method only renders HTML pages, even during POST, therefore we should use `@login_required` here